.github/workflows/build.yaml

```diff
@@ -0,0 +1,27 @@
+name: characters-counter-build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Run checks
+        run: |
+          isort --check --diff . 
```

Comment:
Please separate one check per one - name step. For the sake of atomicy

Reply:
Done!